Link to StatProfileHTML in Profile docs (#32650)
JuliaLang/julia: doc/src/manual/profile.md

julia> @profile myfunc()
```

> To see the profiling results, there are several graphical browsers

Reviewer: ```suggestion
To see the profiling results, there are several graphical browsers.
```

One "family" of visualizers is based on [FlameGraphs.jl](https://github.com/timholy/FlameGraphs.jl), with each family member providing a different user interface:
- [Juno](https://junolab.org/) is a full IDE with built-in support for profile visualization
- [ProfileView.jl](https://github.com/timholy/ProfileView.jl) is a stand-alone visualizer based on GTK
- [ProfileVega.jl](https://github.com/davidanthoff/ProfileVega.jl) uses VegaLight and integrates well with Jupyter notebooks
- [StatProfilerHTML](https://github.com/tkluck/StatProfilerHTML.jl) produces HTML and presents some additional summaries, and also integrates well with Jupyter notebooks
- [ProfileSVG](https://github.com/timholy/ProfileSVG.jl) renders SVG

An entirely independent approach to profile visualization is [PProf.jl](https://github.com/vchuravy/PProf.jl), which uses the external `pprof` tool.

> available, but here we'll use the text-based display that comes with the standard library:

Reviewer: ```suggestion

Here, though, we'll use the text-based display that comes with the standard library:
```

julia> @profile myfunc()
```

To see the profiling results, there are several graphical browsers

> including [ProfileView.jl](https://github.com/timholy/ProfileView.jl) and [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML)

Author: ```suggestion
including [ProfileView.jl](https://github.com/timholy/ProfileView.jl), [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML), and [PProf.jl](https://github.com/vchuravy/PProf.jl).
```

> including [ProfileView.jl](https://github.com/timholy/ProfileView.jl), [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML), and [PProf.jl](https://github.com/vchuravy/PProf.jl).

Reviewer: ```suggestion
One "family" of visualizers is based on [FlameGraphs.jl](https://github.com/timholy/FlameGraphs.jl), with each family member providing a different user interface:
- [Juno](https://junolab.org/) is a full IDE with built-in support for profile visualization
- [ProfileView.jl](https://github.com/timholy/ProfileView.jl) is a stand-alone visualizer based on GTK
- [ProfileVega.jl](https://github.com/davidanthoff/ProfileVega.jl) uses VegaLight and integrates well with Jupyter notebooks
- [StatProfilerHTML](https://github.com/tkluck/StatProfilerHTML.jl) produces HTML and presents some additional summaries, and also integrates well with Jupyter notebooks
- [ProfileSVG](https://github.com/timholy/ProfileSVG.jl) renders SVG

An entirely independent approach to profile visualization is [PProf.jl](https://github.com/vchuravy/PProf.jl), which uses the external `pprof` tool.
```

Reviewer: I wasn't even sure it's worth mentioning ProfileSVG, feel free to delete it if you want.
